### PR TITLE
add ptz to header list and added new version (3.0.1)

### PR DESCRIPTION
**Pull request by paulo-coutinho**
Specify library name and version:  **date/all**

This is also a good place to share with all of us **why you are submitting this PR** (specially if it is a new addition to ConanCenter): is it a dependency of other libraries you want to package? Are you the author of the library? Thanks!

---

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by uilianries — APPROVED** [review]

File: recipes/date/all/conanfile.py [hunk 1]
```diff
@@ -87,6 +89,7 @@ def package(self):
             dst = os.path.join("include", "date")
             self.copy(pattern="date.h", dst=dst, src=src)
             self.copy(pattern="tz.h", dst=dst, src=src)
+            self.copy(pattern="ptz.h", dst=dst, src=src)
             self.copy(pattern="iso_week.h", dst=dst, src=src)
             self.copy(pattern="julian.h", dst=dst, src=src)
             self.copy(pattern="islamic.h", dst=dst, src=src)
```
Comment (prince-chrismc) [inline recipes/date/all/conanfile.py:95]: I feel like this is easier 🌮 

```suggestion
            for file in ["date.h", "tz.h", "ptz.h", "iso_week.h", "julian.h", "islamic.h"]:
                self.copy(pattern=file, dst=dst, src=src)
```

**Review by prince-chrismc — APPROVED** [review]

**Comment by prince-chrismc** [comment]
Please update the title to include the new version add please =)

**Review by SSE4 — APPROVED** [review]

File: recipes/date/all/conanfile.py (hunk 1 above)
Comment (paulo-coutinho) [reply recipes/date/all/conanfile.py:95]: Hi. This is required? Because will reset reviews, build etc.

Comment (paulo-coutinho) [reply recipes/date/all/conanfile.py:95]: I can add it after this PR, because this PR is required to use this library correctly. Can we do in this way?

Comment (prince-chrismc) [reply recipes/date/all/conanfile.py:95]: No required, I approved! hopefully next time this recipe gets updated ... there's a few out of date conventions

**Comment by prince-chrismc** [comment]
Unresolved graph? 😖 

You'll need to retrigger CI, close the pr wait 10s and then re-open it 🔁

**Comment by paulo-coutinho** [comment]
> Unresolved graph? 😖
> 
> You'll need to retrigger CI, close the pr wait 10s and then re-open it 🔁

done

**Comment by paulo-coutinho** [comment]
> > Unresolved graph? 😖
> > You'll need to retrigger CI, close the pr wait 10s and then re-open it 🔁
> 
> done

all green again

**Comment by prince-chrismc** [comment]
CI should merge it soon =)

File: recipes/date/all/conanfile.py (hunk 1 above)
Comment (uilianries) [reply recipes/date/all/conanfile.py:95]: at university, on 1st semester, there was an exercise about for loop, the idea was print from 1 to 10 using C language. A colleague preferred to write printf 10 times, each line for a single number ... don't try it at home.

**Comment by paulo-coutinho** [comment]
> CI should merge it soon =)

I think that need more one reviewer, no?

**Comment by prince-chrismc** [comment]
Nop, CCI rules do not exactly work on GitHub https://github.com/conan-io/conan-center-index/blob/master/docs/review_process.md#getting-your-pull-request-reviewed

**Pull request merged**